cmd/client.go

```diff
@@ -265,6 +265,66 @@ func (cli *Client) AddBridge(c *clipkg.Context) error {
 	return cli.deserializeResponse(resp, &bridge)
 }
 
+// GetBridges returns all bridges.
+func (cli *Client) GetBridges(c *clipkg.Context) error {
+	cfg := cli.Config
+	requestURI := cfg.ClientNodeURL + "/v2/bridge_types"
+
+	page := 0
+	if c != nil && c.IsSet("page") {
+		page = c.Int("page")
+	}
+
+	var links jsonapi.Links
+	var bridges []models.BridgeType
+	err := cli.getPageOfBridges(requestURI, page, &bridges, &links)
+	if err != nil {
+		return err
+	}
+	return cli.errorOut(cli.Render(&bridges))
+}
+
+func (cli *Client) getPageOfBridges(requestURI string, page int, bridges *[]models.BridgeType, links *jsonapi.Links) error {
```

Comment:
This function is almost identical to `getPageOfJobSpecs`, except for the deserialization/return type. Let's pull it out into a reusable helper function.

Reply:
I could merge `GetJobSpecs` with `GetBridges` and `ShowJobSpec` with `ShowBridge` and then use switch statements, not sure if it'd be much cleaner but it would get rid of some duplication